src/components/network/config-tables.js

```diff
@@ -918,13 +918,13 @@ export const TABLES_DEFINITIONS = {
                 canBeInvalidated: true,
             },
             {
-                id: 'ConstantActivePower',
+                id: 'ConsigneActivePower',
```

Comment:
use 'TargetP' and 'TargetQ' existing translation keys

Reply:
it's fixed.